src/components/content/monitor/MonitorChart.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import React from 'react';
+import EChartsReact from 'echarts-for-react';
+import 'echarts/lib/chart/bar';
+import '../../../styles/monitor.css';
+import { Col, Row } from 'antd';
+import moment from 'moment';
+import { MetricProps } from './metricProps';
+
+function MonitorChart({ monitorType, metricProps }: { monitorType: string; metricProps: MetricProps[] }): JSX.Element {
+    interface Option {
```

Comment:
for each of the following field in this interface, can we use the actual types instead of just "object" here. Idealy the d.ts file of echarts package must have all types defined and we should be able to use them.

Reply:
fixed.